website/docs/desktop-testing/Electron.md

```diff
@@ -87,10 +126,10 @@ ipcMain.handle('wdio-electron', () => {
 The custom API can then be called in a spec file:
 
 ```ts
-const someValue = await browser.electron.api('wow'); // default
-const someValue = await browser.electron.myCustomAPI('wow'); // configured using `customApiBrowserCommand`
+const someValue = await browser.electron.api('wow');
+console.log(someValue); // outputs: "such api"
 ```
 
 ## More Information
 
-You can learn more about how to configure the [`wdio-electron-service`](https://www.npmjs.com/package/wdio-electron-service) in the [service docs](/docs/wdio-electron-service).
+You can learn more about how to configure the [`wdio-electron-service`](https://www.npmjs.com/package/wdio-electron-service) in the [service docs](/docs/wdio-electron-service). Make sure to check out our [Electron boilerplate project](https://github.com/webdriverio/electron-boilerplate/actions/runs/6490587562/job/17626575914) that showcases how to intergate WebdriverIO in an example application.
```

Comment:
Do you mean to link to the CI output here?

Reply:
Oh, good catch!

Reply:
fixed: https://github.com/webdriverio/webdriverio/commit/52cd25c36db38b47a17c9ff962c138ddf25b26ce